chapters/extensions/VK_KHR_image_format_list.md

```diff
@@ -0,0 +1,7 @@
+# VK_KHR_image_format_list
+
+> Promoted to core in Vulkan 1.2
+
+Some implementations have cases where they can do optimizations for certain formats. The issue comes when an application sets `VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT` in `VkImageCreateInfo` and there is no way the implementation knows if it's one of the formats to apply an implementation dependent optimization too. By using the `VkImageFormatListCreateInfo` struct and listing the possible formats the implementation only needs to check against the subset of formats listed and still enable an implementation dependent optimization.
```

Comment:
The language isn't great. I suggest borrowing from the extension spec overview - maybe: 

"On some implementations, setting the VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT on image creation can
cause access to that image to perform worse than an equivalent image created without
VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT because the implementation does not know what view formats
will be paired with the image. This may force the implementation to disable (view) format-specific optimizations such as lossless image compression. If the `VkImageFormatListCreateInfo` struct used to explicitly list the view formats the image may be paired with, the implementation may be able to enable format-specific optimization in additional cases."